src/core_modules/capture-core-utils/date/date.utils.js

```diff
@@ -1,17 +1,25 @@
 // @flow
-import moment from '../moment/momentResolver';
+import { moment } from '../moment';
 
 export const displayTypes = {
     SHORT: 'short',
     MEDIUM: 'medium',
     LONG: 'long',
 };
 
-export function getMonthName(monthNumber: number) {
-    const date = moment([0, monthNumber - 1]);
-    return date.format('MMMM');
+/**
```

Comment:
Thanks for writing a comment that would explain the function here!! 🏅

Reply:
👍